docs/DafnyRef/Modules.md

```diff
@@ -104,7 +104,7 @@ of a single implicit unnamed global module.
 
 As described in the previous section, module declarations can be nested.
 It is also permitted to declare a nested module _outside_ of its
-"enclosing" module. So instead of
+"containing" module. So instead of
```

Comment:
```suggestion
"parent" module. So instead of
```
Maybe?

Reply:
The RM used 'parent' for inheritance; 'containing' works for syntactic containment